test/web-platform-tests/to-run.yaml

```diff
@@ -0,0 +1,658 @@
+formAction_document_address.html: [fail, Unknown]
+formaction.html: [fail, Unknown]
+
+---
+
+DIR: html/semantics/forms/resetting-a-form
+
+reset-event.html: [fail, Unknown]
+reset-form-event-realm.html: [fail, Unknown]
+reset-form.html: [fail, Unknown]
+
+---
+
+DIR: html/semantics/forms/the-button-element
+
+button-activate.html: [fail, Unknown]
+button-events.html: [fail, Unknown]
+button-validation.html: [fail, Unknown]
+
+---
+
+DIR: html/semantics/forms/the-form-element
+
+form-action-submission-with-base-url.html: [timeout, Unknown]
+form-action-submission.html: [timeout, Unknown]
+form-autocomplete.html: [fail, Unknown]
+form-elements-filter.html: [fail, Unknown]
+form-elements-interfaces-01.html: [fail, Unknown]
+form-elements-matches.html: [fail, Unknown]
+form-elements-nameditem-01.html: [fail, Unknown]
+form-indexed-element.html: [fail, Unknown]
+form-nameditem.html: [fail, Unknown]
+
+---
+
+DIR: html/semantics/forms/the-input-element
+
+button.html: [fail, Unknown]
+checkbox.html: [fail, Unknown]
+date.html: [fail, Unknown]
+datetime-local.html: [fail, Unknown]
+datetime.html: [fail, Unknown]
+email.html: [fail, Unknown]
+files.html: [fail, Unknown]
+hidden.html: [fail, Unknown]
+input-type-button.html: [fail, Unknown]
+input-type-checkbox.html: [fail, Unknown]
+maxlength.html: [fail, Unknown]
+minlength.html: [fail, Unknown]
+month.html: [fail, Unknown]
+number.html: [fail, Unknown]
+radio-groupname-case.html: [fail, Unknown]
+radio.html: [fail, Unknown]
+range-2.html: [fail, Unknown]
+range.html: [fail, Unknown]
+reset.html: [fail, Unknown]
+text.html: [fail, Unknown]
+time-2.html: [fail, Unknown]
+time.html: [fail, Unknown]
+type-change-state.html: [fail, Unknown]
+valueMode.html: [fail, Unknown]
+week.html: [fail, Unknown]
+
+---
+
+DIR: html/semantics/forms/the-label-element
+
+label-attributes.html: [fail, Unknown]
+labelable-elements.html: [fail, Unknown]
+
+---
+
+DIR: html/semantics/forms/the-option-element
+
+option-label.html: [fail, Our impl is wrong; see comments in HTMLOptionElement-impl.js]
+option-text-recurse.html: [fail, Our impl is wrong; see comments in HTMLOptionElement-impl.js]
+option-value.html: [fail, Our impl is wrong; see comments in HTMLOptionElement-impl.js]
+
+---
+
+DIR: html/semantics/forms/the-select-element
+
+select-validity.html: [fail, Validation is not implemented]
+
+---
+
+DIR: html/semantics/forms/the-textarea-element
+
+cloning-steps.html: [fail, Unknown]
+value-defaultValue-textContent-xhtml.xhtml: [fail, Unknown]
+
+---
+
+DIR: html/semantics/interactive-elements/the-details-element
+
+toggleEvent.html: [fail, The ninth test requires a streaming HTML parser to pass]
+
+---
+
+DIR: html/semantics/scripting-1/the-script-element
+
+async_001.htm: [timeout, Unknown]
+async_002.htm: [timeout, Unknown]
+async_003.htm: [timeout, Unknown]
+async_004.htm: [timeout, Unknown]
+async_005.htm: [timeout, Unknown]
+async_006.htm: [timeout, Unknown]
+async_007.htm: [needs-node8]
```

Comment:
It might be helpful to include the reason for why a newer Node-version is required in the second field, in case more reasons appear in future versions.